aiohttp/web_urldispatcher.py

```diff
@@ -755,6 +756,9 @@ def routes(self):
     def named_resources(self):
         return MappingProxyType(self._named_resources)
 
+    def set_defaults(self, *, allow_head):
```

Comment:
problem is, we currently do not control when handler writes body, so in most cases
developer needs to support `head` explicitly